source/ConfigFiles.h

```diff
@@ -1323,6 +1323,7 @@ class TargetViewConfig {
 	float           combatTextTimeout = 0.5f;							///< Time for combat text to disappear (in seconds)
 
 	// Reference/dummy target
+	bool			showRefTarget = true;								///< Show the reference target?
```

Comment:
Even though I asked to change the name of the config parameter, it's fine to leave this as `showRefTarget` because it matches the other C++ names for reference target variables.